code/datums/keybinding/keybinding.dm

```diff
@@ -22,3 +22,6 @@
 
 /datum/keybinding/proc/up(client/user)
 	return FALSE
+
+/datum/keybinding/proc/intercept_mouse_special(datum/source)
+	return COMSIG_MOB_CLICK_CANCELED
```

Comment:
```suggestion
	return COMSIG_MOB_CLICK_CANCELED

```
Always end files in a new line.

Reply:
I comitted this shit 4 times with definitely trailing new lines idk why it keeps being fucky just kill me